src/App.tsx

```diff
@@ -17,18 +71,32 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                selectedOption={selectedOption}
+                onSelectChange={setSelectedOption}
+                query={query}
+                onQueryChange={setQuery}
+              />
```

Comment:
The method names 'onSelectChange' and 'onQueryChange' do not follow the recommended naming conventions. Consider renaming them to 'handleSelectChange' and 'handleQueryChange' respectively.